docs/store.js

```diff
@@ -437,7 +436,8 @@ var LocalCalendarStorage = function(params) {
             })
             .catch( function(reason) {
                 console.log("getAllCalendarIds catch function running. " + reason);
-                return reason;
+                // returns a new promise which resolves with this value {}
```

Comment:
Comments are about explaining *why* you are doing something, not what it does. You can figure out this comment by reading the docs, but it's not clear why you did it.  E.g. "If there's no index of all calendar ids and their titles, make a new one and use that."